src/OfxNet/Sgml/SgmlParser.cs

```diff
@@ -32,15 +34,39 @@ public SgmlElement Parse(string path, Encoding encoding)
             var line = reader.ReadLine();
             ++this.lineNumber;
 
-            if (string.IsNullOrWhiteSpace(line) == false)
+            if (!string.IsNullOrEmpty(line))
             {
-                this.ProcessLine(line);
+                var tags = line.Split("<").ToList();
+                tags.ForEach(t =>
+                {
+                    if (!string.IsNullOrWhiteSpace(t))
```

Comment:
My preference is explicit comparison with true/false
```suggestion
                    if (string.IsNullOrWhiteSpace(t) == false)
```